pkg/injector/profiling.go

```diff
@@ -41,3 +41,13 @@ func webhookTimeTrack(start time.Time, timeout time.Duration, success *bool) {
 	metricsstore.DefaultMetricsStore.InjectorRqTime.
 		WithLabelValues(fmt.Sprintf("%t", *success)).Observe(elapsed.Seconds())
 }
+
+// Time tracking function for xds certificate issuance.
+// Will report the time taken to issue an xds certificate
+func certXdsIssueTimeTrack(elapsed time.Duration, commonName string, success *bool) {
+	log.Debug().Msgf("Time taken to issue xds certificate %s is %v",
```

Comment:
Lets remove this log, it seems a bit unnecessary if we have the metric being exported already. There is ongoing discussion on not leaking cert details such as CN. 
Can we remove this function and just update the counter in L36 of patch.go?

Reply:
or `trace`, IMO

Reply:
updated